src/Mjolnir.ts

```diff
@@ -138,8 +140,8 @@ export class Mjolnir {
         return this.protections;
     }
 
-    public get redactionHandler(): AutomaticRedactionQueue {
-        return this.redactionQueue;
+    public get unlistedUserRedactionHandler(): UnlistedUserRedactionQueue {
```

Comment:
Likewise, since it's an API, can you document it?

I realize that Mjolnir's source code is not very documented, so let's use any opportunity to fix that :)